automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/FreeTrialSpec.scala

```diff
@@ -5,40 +5,58 @@ import org.broadinstitute.dsde.firecloud.component.{Button, Checkbox, Label, Tes
 import org.broadinstitute.dsde.firecloud.fixture.UserFixtures
 import org.broadinstitute.dsde.firecloud.page.workspaces.WorkspaceListPage
 import org.broadinstitute.dsde.workbench.auth.{AuthToken, TrialBillingAccountAuthToken}
-import org.broadinstitute.dsde.workbench.config.{Credentials, UserPool}
-import org.broadinstitute.dsde.workbench.service.{Google, Orchestration, Thurloe}
+import org.broadinstitute.dsde.workbench.config.{Config, Credentials, UserPool}
 import org.broadinstitute.dsde.workbench.service.test.{CleanUp, WebBrowserSpec}
-import org.scalatest.{BeforeAndAfterEach, FreeSpec, Matchers}
+import org.broadinstitute.dsde.workbench.service.{Google, Orchestration, Thurloe}
+import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach, FreeSpec, Matchers}
 
 import scala.util.Try
 
 
 /**
   * Tests for new user registration scenarios.
   */
-class FreeTrialSpec extends FreeSpec with BeforeAndAfterEach with Matchers with WebBrowserSpec
+class FreeTrialSpec extends FreeSpec with BeforeAndAfterEach with BeforeAndAfterAll with Matchers with WebBrowserSpec
   with UserFixtures with CleanUp with LazyLogging {
 
   val adminUser: Credentials = UserPool.chooseAdmin
   val campaignManager: Credentials = UserPool.chooseCampaignManager
   implicit val authToken: AuthToken = adminUser.makeAuthToken()
   val trialKVPKeys = Seq("trialState", "trialBillingProjectName", "trialEnabledDate", "trialEnrolledDate",
     "trialTerminatedDate", "trialExpirationDate", "userAgreed")
+  val billingProject: String = Config.Projects.default
 
   var testUser: Credentials = _
   var userAuthToken: AuthToken = _
   var subjectId : String = _
 
+  /**
+    * Set up the number of projects necessary for this entire test suite to run.
+    * Currently, that number is 1
+    */
+  val freeTrialProjectsRequired = 1
+
+  override def beforeAll: Unit = {
```

Comment:
I like the idea of this. The downside of this approach is that it will abort the test suite if it fails here. It would be better to make it a fixture and to use that in the testcase.